cluster-scope/overlays/nerc-ocp-prod/externalsecrets/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- default-ingress-certificate.yaml
+- default-api-certificate.yaml
```

Comment:
`external-apps-ingress-certificate` is missing from this list.

Reply:
I guess not, it will show up in https://github.com/OCP-on-NERC/nerc-ocp-config/pull/51#discussion_r956149730

I guess once this is merged, that PR should be rebased.